packages/era/.gitignore

```
@@ -0,0 +1 @@
+*.era
```

Should we also exclude e2s files? Not sure but thought its worth bringing up the question
```suggestion
*.e2s
*.era
```

---

Yeah. Its not used now, but testing/etc might cause an e2s file to be written somewhere. No harm in adding.